packages/stream-tile/src/tile-document.ts

```diff
@@ -279,7 +279,13 @@ export class TileDocument<T = Record<string, any>> extends Stream {
     opts: UpdateOpts = {}
   ): Promise<void> {
     opts = { ...DEFAULT_UPDATE_OPTS, ...opts }
-    const updateCommit = await this.makeCommit(this.api, content, metadata)
+    let signer: CeramicSigner = this.api
```

Comment:
Suggestion: `const signer: CeramicSigner = opts.asDID ? { signer: opts.asDID } : this.api`